spec/aws-record/record/attribute_spec.rb

```diff
@@ -31,6 +31,20 @@ module Record
         end
       end
 
+      context 'default_value' do
+        it 'supports lambdas' do
+          a = Attribute.new(:foo, default_value: -> { Date.today })
+          expect(a.default_value).to eq(Date.today)
```

Comment:
I'm being a little bit pedantic here, but this test can fail if run at exactly the right moment. Can we replace the lambda with a predictable return? Can even be a bit silly:

```ruby
a = Attribute.new(:foo, default_value: -> { 2 + 3 })
expect(a.default_value).to eq(5)
```